examples/mutators/search/simple/.expected/diff.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/resources.yaml b/resources.yaml
```

Comment:
Each package should have its own README file. Look at `examples/template` for what's expected.

Reply:
Done.